assets/js/prebid-server-api.js

```diff
@@ -0,0 +1,22 @@
+var pbs = function() {
+  function getJSON(url, onSuccess, onFailure) {
+    req = new XMLHttpRequest();
+    req.onreadystatechange = function() {
+      if (req.readyState === 4) {
+        if (req.status === 200) {
+          onSuccess(JSON.parse(req.responseText));
```

Comment:
might want a try/catch here in case of malformed content. Although there isn't much you can do to recover so not sure how much it benefits you.

Reply:
No worries... this is still a good idea. We wouldn't want a bad response to stop the page from loading.